src/md/third-party-downloads.md

```diff
@@ -23,4 +18,7 @@ This is a list of 3rd party binary packages based on Apache Apex. The Apache Ape
     - RPM: [CentOS 6](https://ci.bigtop.apache.org/job/Bigtop-trunk-packages/COMPONENTS=apex,OS=centos-6/lastSuccessfulBuild/artifact/output/apex/apex-3.5.0-1.el6.src.rpm), [CentOS 7](https://ci.bigtop.apache.org/job/Bigtop-trunk-packages/COMPONENTS=apex,OS=centos-7/lastSuccessfulBuild/artifact/output/apex/apex-3.5.0-1.el7.centos.src.rpm), [Fedora 25](https://ci.bigtop.apache.org/job/Bigtop-trunk-packages/COMPONENTS=apex,OS=fedora-25/lastSuccessfulBuild/artifact/output/apex/apex-3.5.0-1.fc25.src.rpm), [OpenSUSE 42.1](https://ci.bigtop.apache.org/job/Bigtop-trunk-packages/COMPONENTS=apex,OS=opensuse-42.1/lastSuccessfulBuild/artifact/output/apex/apex-3.5.0-1.src.rpm) 
```

Comment:
I moved these details to the following page:
https://cwiki.apache.org/confluence/display/APEX/ApexBigtop#ApexBigtop-BigtopCIbaseddistributionbinaries

Please link that page instead.